modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/framework/PredefinedSchemaManager.java

```diff
@@ -61,35 +60,23 @@ public class PredefinedSchemaManager implements SqlSchemaManager {
                     schema.getTableNames().stream()
                             .map(schema::getTable)
                             .map(IgniteTable.class::cast)
-                            .collect(Collectors.toMap(IgniteTable::id, Function.identity()))
+                            .collect(CollectionUtils.toIntMapCollector(IgniteTable::id, Function.identity()))
             );
         }
     }
 
-    /** {@inheritDoc} */
-    @Override
-    public SchemaPlus schema(@Nullable String schema) {
-        return schema == null ? root : root.getSubSchema(schema);
-    }
-
     /** {@inheritDoc} */
     @Override
     public SchemaPlus schema(String name, int version) {
```

Comment:
Please note that the **name** may be `null` in the base class and its implementations.